404.qmd

```diff
@@ -0,0 +1,11 @@
+---
+title: "You’ve entered the garden of missing pages 🌱"
+---
+
+Let's get back to greener grounds.
+
+👉 Go to [homepage](/).
+
+![](images/404.svg){fig-alt="An illustration of 404 with trees and mountains"}
```

Comment:
How do you link to an image in a 404 page? I played around with making incorrect links and it seems that root of the 404 page changes depending on where it is - e.g., localhost/guide/incorrect-page looks for an image in the guide dir, while

Reply:
There are some that suggest a full absolute url as the reference, have you tested that?

Reply:
Ah, nice. Thanks!